src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -151,6 +171,11 @@ public static String getIndividualLastName(Entity entity) {
         return entity.getName().getLastName();
     }
 
+    public static String getIndividualFullName(Entity entity) {
+        return entity.getName().getFullName();
```

Comment:
I think this should be getFullNameNoTitle(), we dont use Title as we dont use Hmc title references

Reply:
Done